classes/EbaySynchronizer.php

```diff
@@ -139,23 +140,26 @@ public static function syncProducts($products, $context, $id_lang, $request_cont
 				$data['price_original'] = round($price_original, 2);
 //				$data['price_percent'] = round($clean_percent);
 			} elseif ($price_original > $price)
-				$data['price_original'] = round($price_original, 2);
+                $data['price_original'] = round($price_original, 2);
 			
-			if (isset($data['price_original']))
-				$data['price_percent'] = round( ($price_original - $price) / $price_original * 100.0);
+            if (isset($data['price_original']))
+                $data['price_percent'] = round( ($price_original - $price) / $price_original * 100.0);
 
 			$data['description'] = EbaySynchronizer::_getEbayDescription($product, $ebay_profile, $id_lang);
 
 			// Export to eBay
 			$ebay = EbaySynchronizer::_exportProductToEbay($product, $data, $p['id_ebay_profile'], $ebay_category, $ebay, $date, $context, $id_lang);
+			
+			$handle = fopen(dirname(__FILE__) . '/log-error.txt', 'w');

```

Comment:
The file opened is not used or closed. You must add the missing lines or remove this one.